mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/ChooseSimDialog.java

```diff
@@ -75,8 +76,12 @@ public void onSimSelected(View view) {
     @OnClick(R.id.btn_confirm)
     public void onConfirmClicked() {
         dismiss();
-        if (getActivity() instanceof LinkBankAccountActivity) {
+        if (getActivity() instanceof LinkBankAccountActivity && (selectedSim ==1 || selectedSim == 2)){
```

Comment:
Please insert a space between `)` & `{`. This is leading to a check-style violation. Also before making a pull request, do run the quality checking commands (_refer project README_) locally on your system.

Reply:
Thanks I have updated according to the gradlew check report.